zero_nova/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "nova"
```

Comment:
Maybe we can remove the commits, constants and running instance files from the PR?